tests/common.go

```diff
@@ -9128,6 +9128,20 @@ func KillKvdbMemberUsingPid(kvdbNode node.Node) error {
 	return nil
 }
 
+// CheckIfNodeIsKvdbNode returns true if a node is kvdb node else it returns false
+func CheckIfNodeIsKvdbNode(n node.Node) (bool, error) {
```

Comment:
Can we put this function inside "drivers/node" package?

Reply:
Better to keep it here for now atleast.